app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt

```diff
@@ -151,35 +154,67 @@ class SettingsFragment : Fragment(), SettingsView, AdapterView.OnItemClickListen
                 .setSingleChoiceItems(languages, -1) { dialog, which ->
                     when (which) {
                         0 -> {
-                            mainActivity?.setLocale("en")
+                            activity?.baseContext?.let { it1 -> presenter.setLocale("en", it1) }
```

Comment:
What does `it1` means for?

Reply:
I have used it1 for passing baseContext. Should I use non null asserted(!!) call instead of wrap with?.let{} call?

Reply:
You shouldn't. 
You just need to let us know what `it1` means for.

Reply:
It is unresolved. 
Again, what does `it1` means for?

Reply:
I have used it1 for passing baseContext. I have used wrap with?.let{} call to lead with nullability

Reply:
I see. What I want to know is what `it1` means for?

Reply:
Ok, Now I got it. (No need of using it1) 😅